yorkie-team/yorkie-js-sdk
# Fix delayed synchronization

#### What does this PR do?
When envoy proxy goes down and later comes back up, it used to be that the connection does not get automatically reestablished.

With this fix, JS clients retries to connect to the envoy proxy until it comes back up, and then applies all local changes automatically.


#### How should this be manually tested?
1. Run normally (`docker-compose up` to turn on `mongo`, `envoy` and `yorkie`, and then `npm start` to start the webpack `devServer`)
2. Shut down the envoy proxy (`docker stop envoy`)
3. Make some local changes to the document
4. Restart the envoy proxy (`docker start envoy`)
5. Local changes of each JS clients should be applied automatically

#### Any background context you want to provide?

c.f. #48 

#### What are the relevant tickets?

Fixes #48 

### Checklist
